spec/acceptance/self_service_spec.rb

```diff
@@ -193,6 +193,11 @@ class {'self_service':
 - name: Classifier Configuration Data
   data_hash: classifier_data')
       end
+      it 'if S0031 conditions for false are met' do
+        run_shell('mkdir -p /opt/puppetlabs/server/data/packages/public/2018.1.5/el-7-x86_64-5.5.8')
+        result = run_shell('facter -p self_service.S0031')
+        expect(result.stdout).to match(%r{false})
+      end
```

Comment:
You should remove the directory to clean up and not affect other tests.

```
        run_shell('rm -rf /opt/puppetlabs/server/data/packages/public/2018.1.5')
```

Reply:
Thanks for your Jarret, updated the code.